pkg/apis/anomalydetector/install/install.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Antrea Authors
```

Comment:
I don’t have a strong opinion on this, but imo it would be better have the separate set of APIs in a separate folder and then the sub apis in the same folder, however there is another thought process that all the apis should be in one folder.
lemme know which one works best ?

Reply:
I feel that we should either moving `anomalydetector` into `intelligence` or renaming `intelligence` to `policyrecommendation`, otherwise the package naming strategy is not consistent. I personally prefer the first one, but do not have strong opinion. cc @dreamtalen @wsquan171 for suggestions.

Reply:
We should let `anomalydetector` under `intelligence`, I remember that was what our design when introducing theia manager.

Reply:
Sure, I would like to defer it to #168